origin-dapp/src/css/app.css

```diff
@@ -5490,8 +5546,13 @@ a:not([href]):not([tabindex]).span-link {
   }
 }
 
-/* anything smaller than md */
+/* md - */
```

Comment:
@sparrowDom is this a common way to say "smaller and not including md"? I'm not familiar with this. At a glance, it seems to include `md`.

Reply:
Yup good point. Renaming to `sm -` to avoid confusion